sdk/react/panels/languageSelectionPanel.js

```diff
@@ -117,35 +154,29 @@ var LanguageSelectionPanel = React.createClass({
     var animationStyle = {opacity:this.state.opacity};
 
     if (this.props.selectedLanguage) {
-      var previewText = 
+      var previewText =
         ( <PreviewWidget
             isVisible={hasCC}
             config={this.props.config}>
           </PreviewWidget>
         );
     }
-  
+
     return (
       <Animated.View style={[styles.panelContainer, panelStyles.panel, animationStyle]}>
-        {this.renderHeader()}
-        <ToggleSwitch
-          switchOn={hasCC}
-          areClosedCaptionsAvailable={this.props.languages.length > 0}
-          onValueChanged={(value)=>this.onSwitchToggled(value)}
-          switchOnText={Utils.localizedString(this.props.config.locale, "On", this.props.config.localizableStrings)}
-          switchOffText={Utils.localizedString(this.props.config.locale, "Off", this.props.config.localizableStrings)}
-          config={this.props.config}>
-        </ToggleSwitch>
-        <ResponsiveList
-          horizontal={renderHorizontal}
-          data={this.props.languages}
-          itemRender={this.renderItem}
-          width={this.props.width}
-          height={itemPanelHeight}
-          itemWidth={160}
-          itemHeight={88}>
-        </ResponsiveList>
-        {previewText}
+        <ScrollView>
```

Comment:
I think the ScrollView shouldn't wrap the header so it is always visible. Although I can see why you did it, is it because on smaller screens there is not enough space?

What do you think about having the `renderHeader` outside of the `ScrollView`?